platform/posix/utest/mocks/time_api.h

```diff
@@ -0,0 +1,17 @@
+#ifndef TIME_API_
+#define TIME_API_
+
+#include <time.h>
+
+/* Get current value of clock CLOCK_ID and store it in TP.  */
```

Comment:
nit: rename `tp` to something longer (e.g. `timepoint`) so that we don't have to add a 2-character string which could easily be a misspelled word to the lexicon.  The parameter's name doesn't have to match the original definition, only the type.